python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -28,22 +28,27 @@ def _wrap_image_model(model, examples, model_task, is_function):
     """
     _wrapped_model = model
     if model_task == ModelTask.IMAGE_CLASSIFICATION:
-        _wrapped_model = WrappedImageClassificationModel(model)
+        if str(type(model)).endswith("fastai.learner.Learner'>"):
```

Comment:
Is there a better way to find whether a model is of fastai type? Maybe use `isinstance` with some known base class from fastai?

Reply:
Their learner class wraps all pytorch models:

https://docs.fast.ai/learner.html#learner

Here I try to avoid importing fastai.  We could still add special try/catch around the fastai imports to check if it is installed, and then use the isinstance check.  However, using the string check is more straightforward and easier I think.